CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+0.7.1 / 2017-12-04
```

Comment:
The version can't be the same as the previous one. The date will need changing to today too.